doc/api/cli.md

```diff
@@ -751,6 +751,11 @@ threadpool by setting the `'UV_THREADPOOL_SIZE'` environment variable to a value
 greater than `4` (its current default value). For more information, see the
 [libuv threadpool documentation][].
 
+### `NODE_PENDING_PIPE_INSTANCES=instances`
```

Comment:
Can you add an YAML `added:` block, similar to other items in this file?

Reply:
Also: it seems these sections are sorted alphabetically, so this one needs to be placed after the `NODE_PENDING_DEPRECATION=1`.

Reply:
@addaleax Though `NODE_PENDING_DEPRECATION` could be documented here, it's "added" long time ago rather than a new feature. Maybe we should keep the document here without `added:` block?

> Also: it seems these sections are sorted alphabetically, so this one needs to be placed after the NODE_PENDING_DEPRECATION=1.

Done!